### PR TITLE
UI global setting service

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the  added UI for services in the global setting menu
[#6211](https://github.com/open-metadata/OpenMetadata/issues/6211)
#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/180989746-7e9b0abb-ac31-4a5d-a691-98b12bbeda4d.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+  };
+  const getOptionalFields = (service: DataService): JSX.Element => {
+    switch (serviceName) {
+      case ServiceCategory.MESSAGING_SERVICES: {
+        const messagingService = service as MessagingService;
+
+        return (
+          <div className="tw-mb-1 tw-truncate" data-testid="additional-field">
+            <label className="tw-mb-0">Brokers:</label>
+            <span
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx:76]: Can this be part of utils? Or it's specific to settings page?

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+  serviceData,
+  serviceName,
+  paging,
+  currentPage,
+  onPageChange,
+}: ServicesProps) => {
+  const { isAdminUser } = useAuth();
+  const { isAuthDisabled } = useAuthContext();
+  const history = useHistory();
+  const goToAddService = () => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx:65]:
```suggestion
  const handleAddServiceClick = () => {
```

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+                  data-testid="service-card">
+                  <div className="tw-flex tw-flex-col tw-justify-between tw-truncate">
+                    <div>
+                      <Link
+                        to={getServiceDetailsPath(service.name, serviceName)}>
+                        <button>
+                          <h6
+                            className="tw-text-base tw-text-grey-body tw-font-medium tw-text-left tw-truncate tw-w-48"
+                            data-testid={`service-name-${getEntityName(
+                              service as unknown as EntityReference
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx:179]: Can we type this batter?

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+                theme="primary"
+                variant="contained"
+                onClick={goToAddService}>
+                Add New Service
+              </Button>
+            </NonAdminAction>
+          </div>
+          <div
+            className="tw-grid xl:tw-grid-cols-4 tw-grid-cols-2 tw-gap-4 tw-mb-4"
+            data-testid="data-container">
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx:165]: <Space> will be a good fit here: https://ant.design/components/space/#header

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+                      data-testid="service-icon">
+                      {getServiceLogo(service.serviceType || '', 'tw-h-8')}
+                    </div>
+                  </div>
+                </div>
+              </Card>
+            ))}
+          </div>
+          {!isNil(paging.after) ||
+            (!isNil(paging.before) && (
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx:225]: Let's create one util function to check this. Like 

```
showPagination(paging: Paging) => boolean
```

File: openmetadata-ui/src/main/resources/ui/src/pages/services/ServicesPage.tsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+  useEffect(() => {
+    const path = location.pathname.split('/');
+    const serviceType: keyof ServiceCategory = path[
+      path.length - 1
+    ] as keyof ServiceCategory;
+    setServiceName(SERVICE_CATEGORY[serviceType]);
+    setCurrentPage(1);
+    getServiceDetails(SERVICE_CATEGORY[serviceType]);
+  }, [location.pathname]);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/services/ServicesPage.tsx:73]: Can you explain this to me? I feel we can replace this with router

File: openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/Services/Services.tsx:76]: This is specific to the service page

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**